cocos/game/splash-screen.ts

```diff
@@ -340,7 +340,8 @@ export class SplashScreen {
         region.texExtent.width = this.bgImage.width;
         region.texExtent.height = this.bgImage.height;
         region.texExtent.depth = 1;
-        device.copyTexImagesToTexture([this.bgImage], this.bgTexture, [region]);
+
+        device.copyImagesToTexture([this.bgImage], this.bgTexture, [region]);
```

Comment:
Why change the code?

Reply:
The type of the parameter has changed.
`copyTexImagesToTexture` may need to be deprecated because of the dependency on the dom-related interface.